__tests__/account.test.ts

```diff
@@ -147,4 +126,60 @@ describe('deploy and test Wallet', () => {
 
     expect(await account.verifyMessage(typedDataExample, signature)).toBe(true);
   });
+
+  describe('new deployed account', () => {
+    let newAccount: Account;
+
+    beforeAll(async () => {
+      const starkKeyPair = ec.genKeyPair();
+      const starkKeyPub = ec.getStarkKey(starkKeyPair);
+
+      const accountResponse = await defaultProvider.deployContract({
```

Comment:
maybe we can pull out all appearences of defaultProvider and have it defined like
```
export const provider = new Provider({ network: "alpha-goerli" })
```
in some file we always import in all tests? would make the transition to devnet easier

Reply:
Agreed 100%